src/ibex_bluesky_core/callbacks/__init__.py

```diff
@@ -66,6 +67,8 @@ def __init__(
         live_fit_logger_output_dir: str | PathLike[str] | None = None,
         live_fit_logger_postfix: str = "",
         human_readable_file_postfix: str = "",
+        live_fit_update_every: int | None = 1,
+        live_plot_show_every_event: bool = True,
```

Comment:
I don't really like this naming - `live_plot_show_every_event` sounds like "dont show every event, just some of them". 

would `live_plot_update_on_every_event` be clearer?